{"name": "注册手机号获取验证码", "status": "failed", "statusDetails": {"message": "AssertionError: assert '15397919581' in ['距离上次发送未超过1分钟']", "trace": "self = <test_case.usercenter.test_user.TestUser object at 0x00000208229639A0>\ndatas = {'expected': '15397919581', 'mobile': '15397919581'}\n\n    @allure.story(\"用户注册获取验证码\")\n    @allure.title(\"注册手机号获取验证码\")\n    @pytest.mark.parametrize(\"datas\", data['test_send_code'])\n    def test_send_code(self, datas):\n        result = send_code(datas['mobile'])\n        assert result.success is True\n>       assert datas['expected'] in result.body['mobile']\nE       AssertionError: assert '15397919581' in ['距离上次发送未超过1分钟']\n\ntest_case\\usercenter\\test_user.py:32: AssertionError"}, "attachments": [{"name": "stdout", "source": "c3a8bdc7-3019-44cf-8b9c-b90991088460-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "datas", "value": "{'mobile': '15397919581', 'expected': '15397919581'}"}], "start": 1671380864906, "stop": 1671380864985, "uuid": "61c1c62b-4748-4d12-a45d-ae493b9f94c8", "historyId": "a1b5580102f67c8a9a030218a443f6c6", "testCaseId": "f1a3dd11e8049b14673a1b3ecac76225", "fullName": "test_case.usercenter.test_user.TestUser#test_send_code", "labels": [{"name": "feature", "value": "用户中心模块"}, {"name": "story", "value": "用户注册获取验证码"}, {"name": "parentSuite", "value": "test_case.usercenter"}, {"name": "suite", "value": "test_user"}, {"name": "subSuite", "value": "TestUser"}, {"name": "host", "value": "xiaoma"}, {"name": "thread", "value": "303052-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.usercenter.test_user"}]}